core/sequencer/server_test.go

```diff
@@ -42,12 +44,21 @@ func fakeMetric(_ string) {}
 // fakeLogs are indexed by logID, and nanoseconds from time 0
 type fakeLogs map[int64][]mutator.LogMessage
 
+func micros(t time.Time) int64 {
+	return t.Sub(zero).Microseconds()
+}
+
+func init() {
+	initMetrics.Do(func() { createMetrics(monitoring.InertMetricFactory{}) })
+}
+
 func (l fakeLogs) ReadLog(ctx context.Context, directoryID string, logID int64, low, high time.Time,
 	batchSize int32) ([]*mutator.LogMessage, error) {
+	log.Printf("ReadLog(%v, %v, %v, %v)", directoryID, logID, low.UnixNano(), high.UnixNano())
```

Comment:
What's the policy on logging via `log`, `glog`, and the test logger?

Reply:
Always glog in server. t.Log in tests. This was left over debugging prints